epoxy-adapter/src/main/java/com/airbnb/epoxy/AsyncEpoxyController.java

```diff
@@ -0,0 +1,41 @@
+package com.airbnb.epoxy;
+
+import android.os.Handler;
+
+import static com.airbnb.epoxy.EpoxyAsyncUtil.MAIN_THREAD_HANDLER;
+import static com.airbnb.epoxy.EpoxyAsyncUtil.getAsyncBackgroundHandler;
+
+/**
+ * A subclass of {@link EpoxyController} that makes it easy to do model building and diffing in
+ * the background.
+ * <p>
+ * See https://github.com/airbnb/epoxy/wiki/Epoxy-Controller#asynchronous-support
+ */
+public abstract class AsyncEpoxyController extends EpoxyController {
+
+  /**
+   * A new instance that does model building and diffing asynchronously.
+   */
+  public AsyncEpoxyController() {
+    this(true);
+  }
+
+  /**
+   * @param enableAsync True to do model building and diffing asynchronously, false to do them
+   *                    both on the main thread.
+   */
+  public AsyncEpoxyController(boolean enableAsync) {
+    this(true, true);
+  }
```

Comment:
`enableAsync` is unused.

Reply:
@baoti whoops! thanks a lot for catching that